framework_tvb/tvb/interfaces/web/templates/jinja2/project/structure.html

```diff
@@ -29,6 +31,12 @@ <h3>Data Structure</h3>
                                 title="Import ">Upload
                         </button>
                     </li>
+                    <li>
+                        <button tabindex='50' type="submit" class="action action-upload"
+                                onclick="importTumorDataset({{ project.id }})"
```

Comment:
From the server, we could send here the tumor importer algorithm ID in the same manner as we send the project ID. Or we could also send the whole link for the Ajax call (this template is rendered by `ProjectController.editstructure`)

Reply:
Thanks for the idea, I added the algorithm id there.